google-datacatalog-connectors-commons/src/google/datacatalog_connectors/commons/prepare/datacatalog_strings_helper.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import six
+
+
+class DataCatalogStringsHelper:
+    # String field values are limited to 2000 bytes size when encoded in UTF-8.
+    __TRUNCATED_STRING_SUFFIX_CHARS_LENGTH = 3
+    __UTF8_CHARACTER_ENCODING = 'UTF-8'
+
+    @classmethod
+    def truncate_string(cls, value, max_length):
+        """
+        String field values are limited by Data Catalog API at 2000 chars
```

Comment:
Please delete the first phrase as it only matters to the former `_set_string_field` method.